docs/Release_notes.md

```diff
@@ -1,4 +1,16 @@
 V2X-Hub Release Notes
+---------------------------------
+Version 7.3.0, released June 13th, 2022
+--------------------------------------------------------
+**Summary:**
+V2X Hub release 7.3.0 includes added functionality for subscribing to FLIR camera for Pedestrian tracking and PSM broadcast to vehicles in the Pedestrian Plugin.
+
+Bug fixes in this release:
```

Comment:
We only need to document issues that were entered by external users, fixes for any issues found in previous releases or during this release testing for old functionality; issues found during our internal testing for the new functionality being released need not be included, please remove 374 and 383.  It also keeps the release notes short and clean.  I would instead expand on the FLIR interface that was implemented if there is a reference we can include to what was implemented.